public/data/Axorax.json

```
@@ -0,0 +1,13 @@
+{
+ "name": "Aayan Abdullah",
+ "location": "Bangladesh",
+ "bio": "I like to do stuff.",
+ "avatar": "https://github.com/axorax.png",
+ "portfolio": "https://github.com/axorax",
+ "skills": ["web development"],
+ "social": {
+   "GitHub": "https://github.com/axorax",
+   "Twitter": "https://twitter.com/axorax_",
+   "LinkedIn": "https://www.linkedin.com/in/axorax"
+ }
+},
```

```suggestion
}
```
Remove the last comma.